docs/fast_data/inputs_and_outputs.md

```diff
@@ -83,100 +98,108 @@ Delete operation:
 {
   "key": {
     "USER_ID": 123,
-    "FISCAL_CODE": "ABCDEF12B02M100O"
+    "TAX_CODE": "ABCDEF12B02M100O"
   },
   "value": null,
   "timestamp": "1234556789",
-  "offset": "100"
+  "offset": 100
 }
 ```
 
 #### Oracle Golden Gate
 
 ```yaml
-asyncapi: 2.4.0
+asyncapi: 2.6.0
 info:
-  title: Golden Gate Data Change Producer
+  title: Data Change API
   version: "1.0.0"
 channels:
-  GoldenGateDataChangeChannel:
+  GoldenGate:
     publish:
       message:
-        name: golden gate data change
+        name: Golden Gate data change message
         payload:
           type: object
-          additionalProperties: false
+          required:
+            - key
+            - value
+            - timestamp
+            - offset
           properties:
-            key: {}
+            key: 
+              type: string
+              description: String of the primary keys values joined by underscores
+              examples:
+                - pkValue1_pkValue2
             value:
               type: object
               additionalProperties: false
+              required:
+                - op_type
+                - pos
```

Comment:
pos could be not required if internal counter is timestamp

Reply:
I guess the internal counter is always defined via timestamp. `pos` will be anyway included in a message sent via Golden Gate CDC. I think is fine to leave it there.